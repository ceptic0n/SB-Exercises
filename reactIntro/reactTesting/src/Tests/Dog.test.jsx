import React from "react";
import { render } from "@testing-library/react";
import Dog from "../Components/Dog";

it("should render", () => {
    expect(<Dog name="Rusty" isAdopted={true}/>);
})

it("Should match snapshop", () =>{
    const { asFragment } = render(<Dog name="Rusty" isAdopted={true}/>);
    expect(asFragment()).toMatchSnapshot();
})