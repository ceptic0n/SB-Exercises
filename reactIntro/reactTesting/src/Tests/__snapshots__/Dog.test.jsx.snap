// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Should match snapshop 1`] = `
<DocumentFragment>
  <div
    class="Dog"
  >
    <h1>
      Rusty
    </h1>
    <p>
      Rusty has been adopted
    </p>
  </div>
</DocumentFragment>
`;
